### ISSUE TITLE
Tune model with own dataset

### ISSUE THREAD

**Issue by bpeck81** [eldar/pose-tensorflow#86]
Hi eldar, thank you for your work. I noticed that the model accuracy declines with photos in darker settings. I would like to improve the model accuracy by introducing darker data from my application and am wondering the best way to do that in an efficient manner. Could you suggest if I should freeze earlier layers of the network or add new convolutional laters while training with my own data. Or will I have to retrain the entire resnet.

Thanks

**Comment by eldar** [comment]
Hi @bpeck81 and sorry for the late reply. If this is still relevant for you, I would recommend to fine-tune the entire network, but use fewer iterations. Freezing layers is not necessary, but that could be tried. In order to do that you will have to modify the code and explicitly pass the list of variables that you want to optimize to the optimizer in the training script. Closing this one, as this is not really an issue. You can send me an email if you still have questions.

Cheers,
Eldar.
